GAE/src/org/waterforpeople/mapping/dataexport/ExportUtils.java

```diff
@@ -0,0 +1,97 @@
+
+package org.waterforpeople.mapping.dataexport;
+
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+import org.apache.log4j.Logger;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.Row;
+
+import com.gallatinsystems.common.util.StringUtil;
+
+public class ExportUtils {
```

Comment:
Pedantic comments welcome :+1: 

What would be a better name? `ImportExportUtils`? The general idea is to share as much as possible between Import and Export for things that needs to be kept in sync (md5 calculation, date formatting etc.)